package.json

```diff
@@ -55,7 +55,7 @@
     "proxyquire": "^2.0.1",
     "should": "^11.2.1",
     "should-http": "^0.1.0",
-    "sinon": "^2.3.4",
+    "sinon": "4.1.3",
```

Comment:
@dwaynebailey - any reason to lock `sinon` to 4.1.3?

Reply:
The version of `sinon-express-mock` that we use is locked to a specific version.

```
~/dev/moj/repos/ccd-api-gateway (RDM-2742-yarn_check)$ yarn check
yarn check v1.13.0
error "sinon-express-mock#sinon@4.1.3" doesn't satisfy found match of "sinon@4.1.6"
error Found 1 errors.
```

Reply:
I've added the explanation to the commit message